runsc/boot/fs.go

```diff
@@ -219,6 +224,8 @@ func mountFlags(opts []string) fs.MountSourceFlags {
 			mf.NoAtime = true
 		case "noexec":
 			mf.NoExec = true
+		case "bind", "rbind":
+			mf.Bind = true
```

Comment:
In this case you don't need to set anything to `mf`. Just add a comment that "bind" in options is converted to a 9P mount.